pyrometheus/codegen/python.py

```diff
@@ -133,9 +134,14 @@ class Thermochemistry:
     .. automethod:: get_mix_molecular_weight
     .. automethod:: get_concentrations
     .. automethod:: get_mole_fractions
+    .. automethod:: get_mass_average_property
+    .. automethod:: get_mole_average_property
```

Comment:
Why are mole-averaged properties needed?

Reply:
Tbh, I dont recall exactly. I thought it was important when I started but doesn't look like it anymore. I will double check. In case we remove this will also eliminate some comments you made below.

I will get back on this soon and give an update.

Reply:
Ok, happy to keep them. I just wasn't sure if mole-based properties are needed anywhere in the formulation.